.github/workflows/build-docker.yaml

```diff
@@ -1,48 +1,63 @@
-name: Build scpca-tools image
+name: Build scpca-tools images
 
 on:
   push:
     branches:
       - main
     tags:
-      - 'v*'
+      - "v*"
   pull_request:
     branches:
       - main
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: alexslemonade/scpca-tools
+  REGISTRY_USER: alexslemonade
 
 jobs:
   build_and_push:
     runs-on: ubuntu-22.04
     permissions:
       contents: read
       packages: write
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - image_name: scpcatools
+            dockerfile: Dockerfile
+          - image_name: scpcatools-slim
+            dockerfile: scpcatools_slim.Dockerfile
+          - image_name: scpcatools-reports
+            dockerfile: scpcatools_reports.Dockerfile
+          - image_name: scpcatools-scvi
+            dockerfile: scpcatools_scvi.Dockerfile
+          - image_name: scpcatools-seurat
+            dockerfile: scpcatools_seurat.Dockerfile
```

Comment:
Are we planning on pulling these out into separate repos rather than use tags?

Reply:
This generates them as separate images with their own base names/repositories. The language a bit confusing since the docker command line refers to the set of identifiers as a "tag", but what I am setting up here is something like: `alexslemonade/<image_name>:<version>`. Before I had it as `alexslemonade/scpcatools:<variant>-<version>`, but after Jackie's suggestion I changed to the more distinct names.

Reply:
Ah okay, thanks for the clarification!